plugin/pkg/admission/defaultingressclass/admission.go

```diff
@@ -125,8 +136,17 @@ func (a *classDefaulterPlugin) Admit(ctx context.Context, attr admission.Attribu
 }
 
 // getDefaultClass returns the default IngressClass from the store, or nil.
-func getDefaultClass(lister networkingv1listers.IngressClassLister) (*networkingv1.IngressClass, error) {
+func (a *classDefaulterPlugin) getDefaultClass(ctx context.Context, lister networkingv1listers.IngressClassLister, ingress *networking.Ingress) (*networkingv1.IngressClass, error) {
 	list, err := lister.List(labels.Everything())
+	if apierrors.IsNotFound(err) {
+		// if not found, informer cache could be stale, try a live lookup
+		ing, err := a.client.Get(ctx, *ingress.Spec.IngressClassName, metav1.GetOptions{})
```

Comment:
`getDefaultClass` is only called when `ingress.Spec.IngressClassName` is nil... this will panic